src/Components/Menu.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import User from "../Asset/user.png";
+import ChatBubble from "../Asset/speech-bubble.png";
+import Setting from "../Asset/setting.png";
+
+import styled from "styled-components";
+
+const MenuBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: rgb(66, 54, 48);
+  width: 5rem;
+  height: 100vh;
+`;
+
+const ImgBox = styled.img`
+  width: 3rem;
+  margin-top: 1rem;
+  display: flex;
+`;
+
+const Menu = () => {
+  return (
+    <>
+      <MenuBox>
+        <Link to="/">
+          <ImgBox src={User} />
+        </Link>
+        <Link to="/chatPage">
+          <ImgBox src={ChatBubble} />
+        </Link>
+        <Link to="/settings">
+          <ImgBox src={Setting} />
+        </Link>
+      </MenuBox>
+    </>
```

Comment:
저도 사실 시맨틱 태그에 대해서 잘 모르지만,,, 알아가 보려고 노력 중이랍니다 :)
내비게이션임을 알려주는 태그는 <nav>라고 한다고 해요!
```suggestion
    <nav>
      <MenuBox>
        <Link to="/">
          <ImgBox src={User} />
        </Link>
        <Link to="/chatPage">
          <ImgBox src={ChatBubble} />
        </Link>
        <Link to="/settings">
          <ImgBox src={Setting} />
        </Link>
      </MenuBox>
    </nav>
```

Reply:
오 이런 디테일 좋네요 시원님

Reply:
시맨틱 태그는 모든 소비자용 어플리케이션에서 아주 중요합니다! 근데 시맨틱 태그를 사용하는 가장 큰 이유가 결국엔 SEO인데, 구글의크롤링 봇을 제외하면 모두 SPA에 대해 크롤링을 지원하지 않다보니 무의미해지는 부분이 없지않아 생기게 됩니다... _next.js를 스터디때 넣어야하려나...._